packages/docusaurus-theme-common/src/hooks/useWindowSize.ts

```diff
@@ -17,13 +17,12 @@ const windowSizes = {
 
 type WindowSize = keyof typeof windowSizes;
 
-const DesktopThresholdWidth = 996;
-
-function getWindowSize() {
+function getWindowSize(desktopThresholdWidth = 996): WindowSize {
```

Comment:
Make this an object param:

```suggestion
function getWindowSize({ desktopThresholdWidth = 996 } = {}): WindowSize {
```

Reply:
Thanks for the feedback! I made the change and updated the docs in the `styling-layout.mdx` page in the section about the CSS media queries.